src/main/java/com/paneedah/weaponlib/WeaponRenderer.java

```diff
@@ -3356,14 +3360,19 @@ public void renderItem(ItemStack weaponItemStack, RenderContext<RenderableState>
 				//GlStateManager.translate(-0.05*test1, 0.01*test1, 0);
 				//GlStateManager.rotate(-10f*test1, 1, 1, 0);
 				//mc.getTextureManager().bindTexture(new ResourceLocation("mw:textures/items/sexmoiv.png"));
-				
-				getBuilder().getModel().render(this.player,
-		                renderContext.getLimbSwing(),
-		                renderContext.getFlimbSwingAmount(),
-		                renderContext.getAgeInTicks(),
-		                renderContext.getNetHeadYaw(),
-		                renderContext.getHeadPitch(),
-		                renderContext.getScale());
+
+				if (getBuilder().getModel() != null) {
+					getBuilder().getModel().render(this.player,
+							renderContext.getLimbSwing(),
+							renderContext.getFlimbSwingAmount(),
+							renderContext.getAgeInTicks(),
+							renderContext.getNetHeadYaw(),
+							renderContext.getHeadPitch(),
+							renderContext.getScale());
+				} else {
+					getBuilder().getBakedModel();
```

Comment:
Add a todo above that:

// Todo: Create rendering base on the render methods above